packages/react/src/components/participant/ParticipantTile.tsx

```diff
@@ -73,6 +73,7 @@ export interface ParticipantTileProps extends React.HTMLAttributes<HTMLDivElemen
   disableSpeakingIndicator?: boolean;
 
   onParticipantClick?: (event: ParticipantClickEvent) => void;
+  placeholders?: { [index: string]: React.ReactNode };
```

Comment:
not a big fan of passing the whole map to each participant tile. 
A pattern where the component can take a single placeholder as a ReactNode seems preferable to me. 
In general the codebase tries to avoid passing of components as props and rather works with slottable components (as children). We're already using those for ParticipantTile as custom tile renderings though, so I'm not totally against an exception here.